pkg/migrations/internal/seq_num.go

```diff
@@ -4,13 +4,13 @@ var (
 	// CurrentDBVersionSeqNum is the current DB version number.
 	// This must be incremented every time we write a migration.
 	// It is a shared constant between central and the migrator binary.
-	CurrentDBVersionSeqNum = 184
+	CurrentDBVersionSeqNum = 185
 
 	// MinimumSupportedDBVersionSeqNum is the minimum DB version number
 	// that is supported by this database.  This is used in case of rollbacks in
 	// the event that a major change introduced an incompatible schema update we
 	// can inform that a rollback below this is not supported by the database
-	MinimumSupportedDBVersionSeqNum = 184
+	MinimumSupportedDBVersionSeqNum = 185
```

Comment:
@dashrews78 might be a dumb question, but: in cases like this where we only replace existing data sets but don't necessarily have schema changes, should this still be bumped or we just leave as-is?

Reply:
@dhaus67 Not a dumb question.  In 4.1, yes we should bump that.  In 4.2 that should evolve such that we only need to set it for changes that break the compatibility of the database with the previous releases software.  But I had to put this piece in now so rollbacks from the future to 4.1 could be prevented if the future made schema changes that 4.1 could not support.  I have tickets to update the README and provide examples as part of my 4.2 work.